chart/templates/secrets/redis-secrets.yaml

```diff
@@ -61,38 +60,10 @@ metadata:
     "helm.sh/hook-weight": "0"
 type: Opaque
 data:
+{{- if $redis_is_deployed_by_chart }}
   connection: {{ (printf "redis://:%s@%s-redis:6379/0" (default $random_redis_password .Values.redis.password) .Release.Name) | b64enc | quote }}
-{{- else if and .Values.redis.password .Values.redis.url  }}
-kind: Secret
-apiVersion: v1
-metadata:
-  name: {{ .Release.Name }}-broker-url
-  labels:
-    release: {{ .Release.Name }}
-    chart: {{ .Chart.Name }}
-    heritage: {{ .Release.Service }}
-  annotations:
-    "helm.sh/hook": "pre-install"
-    "helm.sh/hook-delete-policy": "before-hook-creation"
-    "helm.sh/hook-weight": "0"
-type: Opaque
-data:
-  connection: {{ (printf "redis://:%s@%s:6379/0" .Values.redis.password .Values.redis.url) | b64enc | quote }}
-{{- else if .Values.redis.url }}
-kind: Secret
-apiVersion: v1
-metadata:
-  name: {{ .Release.Name }}-broker-url
-  labels:
-    release: {{ .Release.Name }}
-    chart: {{ .Chart.Name }}
-    heritage: {{ .Release.Service }}
-  annotations:
-    "helm.sh/hook": "pre-install"
-    "helm.sh/hook-delete-policy": "before-hook-creation"
-    "helm.sh/hook-weight": "0"
-type: Opaque
-data:
-  connection: {{ (printf "redis://@%s:6379/0" .Values.redis.url) | b64enc | quote }}
+{{- else }}
+  connection: {{ (printf "%s" .Values.redis.brokerURL) | b64enc | quote }}
```

Comment:
you actually removed a condition that seemed important when you have an url + auth scheme with password
(if you want to pass it within the brokerUrl, this variable
https://github.com/MeilleursAgents/airflow/blob/8e0df75dd27ed095d4b8a4e57adef2994b5a12e2/chart/templates/secrets/redis-secrets.yaml#L41
needs still a priori to be provisioned)?

Reply:
Not sure... let's discuss this tomorrow